### PR TITLE
User/colinli02/106 add ci status badges

### LINKED ISSUE #106: Add CI status badges
### Pending
- [x] CI changes from #100 

### Purpose

Add CI status badges to the README

### Resources

- Example: https://github.com/UBCSailbot/docs/blob/main/README.md
    - File diff: https://github.com/UBCSailbot/docs/pull/75/files
- Documentation: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-workflow-file-name

### PR DESCRIPTION
**Pull request by colinli02**
### Description

- Resolves #106 

### Verification

- Verify by checking CI status badges on [here](https://github.com/UBCSailbot/sailbot_workspace/tree/user/colinli02/106-Add-CI-Status-Badges) 
image

### CODE REVIEW

File: .github/workflows/test_definitions.yml [hunk 1]
```diff
@@ -122,4 +122,4 @@ jobs:
     - name: Dispatch Docs Publish workflow
       # https://github.com/orgs/community/discussions/26323#discussioncomment-3251448
       run: |
-        curl -XPOST -u "${{ secrets.PAT_USERNAME}}:${{secrets.PAT_TOKEN}}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/UBCSailbot/docs/actions/workflows/publish.yml/dispatches --data '{"ref": "main"}'
+        curl -XPOST -u "${{ secrets.PAT_USERNAME}}:${{secrets.PAT_TOKEN}}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/UBCSailbot/docs/actions/workflows/publish.yml/dispatches --data '{"ref": "main"}'
```
Comment (patrick-5546) [inline .github/workflows/test_definitions.yml:125]: lol why do you always remove the empty line at the end of the file? You can check the files changes tab to check that no unintended changes were made

**Review by patrick-5546 — CHANGES_REQUESTED** [review]
Will approve once the comment below is addressed

File: .github/workflows/test_definitions.yml (hunk 1 above)
Comment (colinli02) [reply .github/workflows/test_definitions.yml:125]: Why does it keep doing that? I'm going to check

**Review by patrick-5546 — APPROVED** [review]
I removed the Build Images CI badge as I don't think it's relevant to everyone

**Pull request merged**